torchtext/experimental/datasets/raw/common.py

```diff
@@ -1,6 +1,14 @@
 import torch
 
 
+def check_default_set(data_select, target_select):
+    if isinstance(data_select, str):
+        data_select = (data_select)
+    if not set(data_select).issubset(set(target_select)):
+        raise TypeError('A subset of data selection {} is supported!'.format(target_select))
```

Comment:
Write out what the user supplied as well

Reply:
Added.